spec/models/manageiq/providers/nuage/network_manager_spec.rb

```diff
@@ -73,21 +73,6 @@
         expect { @ems.verify_credentials(:amqp) }.to raise_error(StandardError)
       end
     end
-
-    context 'credentials encoding' do
```

Comment:
We are removing this because it is no longer needed as creds are not part of AMQP URL, right?

Reply:
Correct. Not only creds are not needed anymore as part of AMQP URL, but it's now even an error to include them there, so this test isn't valid anymore.